src/Apps/Comparables/Filter/AttributionClassFilter.tsx

```diff
@@ -0,0 +1,70 @@
+import { FilterState } from "Apps/Collect/FilterState"
+import React from "react"
+
+import { Checkbox } from "@artsy/palette"
+import { SystemProps } from "Artsy/SystemContext"
+
+export const AttributionClassFilter: React.SFC<{
+  filters: FilterState
+  mediator: SystemProps["mediator"]
```

Comment:
Filters and mediator can be pulled from `<Context>` and `<Subscribe>` so that things remain decoupled and props passing isn't needed.

Reply:
This type can be [imported directly](https://github.com/artsy/reaction/blob/master/src/Artsy/SystemContext.tsx#L8): `import { Mediator } from 'Artsy/SystemContext'`